.github/actions/build-tasklist-fe/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Tasklist FE
```

Comment:
❓ Do we want the same for Operate?

Reply:
yes, we do, I think it should be done by Operate team with their sauce. 
I think you wanted to replace mvn with GHA to build Operate FE @pedesen